custom/conf/app.example.ini

```diff
@@ -2541,6 +2541,11 @@ LEVEL = Info
 ;; * no-sanitizer: Disable the sanitizer and render the content inside current page. It's **insecure** and may lead to XSS attack if the content contains malicious code.
 ;; * iframe: Render the content in a separate standalone page and embed it into current page by iframe. The iframe is in sandbox mode with same-origin disabled, and the JS code are safely isolated from parent page.
 ;RENDER_CONTENT_MODE=sanitized
+;;
+;; Whether post-process the rendered HTML content, including:
+;; resolve relative links and image sources, recognizing issue/commit references, escaping invisible characters,
+;; mentioning users, rendering permlink code blocks, replacing emoji shorthands, etc.
+;NEED_POST_PROCESS=false
```

Comment:
a) Should that really be an admin option?
Shouldn't we just always use one approach (either no postprocessing or postprocessing)?
b) Should the default value be `false` or `true`?